setup.py

```diff
@@ -9,10 +9,16 @@
     "pre-commit",
     "black==20.8b1",
 ]
-MPI_DEPENDENCIES = ["mpi4py>=3.0.1"]
-JAX_DEPENDENCIES = ["jax"]
-
-MPIJAX_DEPENDENCIES = ["mpi4jax>=0.2.7"]
+MPI_DEPENDENCIES = ["mpi4py>=3.0.1", "mpi4jax>=0.2.9"]
+BASE_DEPENDENCIES = [
+    "numpy>=1.16",
+    "scipy>=1.5.2",
+    "tqdm>=4.42.1",
```

Comment:
```suggestion
    "tqdm>=4.56.2",
```

Not sure which is the exact minimum version, but with tqdm 4.55.1 I get `AttributeError: 'tqdm' object has no attribute '_time'`, with 4.56.2 it works fine.

Reply:
thanks